script.domoticz.scenes/.gitignore

```diff
@@ -0,0 +1,4 @@
+pysrc
```

Comment:
... remove the WHOLE gitignore file from the submission.

Reply:
Removed